stacks/root.yml

```diff
@@ -98,6 +98,15 @@ Mappings:
       abbreviation: stag
     Production:
       abbreviation: prod
+  ASGScalingMap:
```

Comment:
Made sense to include these numbers "near" the ECSReservationMap, since they're related and should be kept in-sync.

Reply:
Does it make any sense to abstract the reservations into like small/medium/large and just literally use the same value references of large for the threshold?

Reply:
I think I've actually figured out a better way to do this.  Instead of ENV-ing it, just have the lambda ask aws for a list of all service-task-defs in the cluster, and pull out the largest CPU/Memory reservations there.  Less configs to keep in-sync.

Reply:
Ooo I like that